### PR TITLE
Document GHA transition (SOFTWARE-4343)

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

**Review by matyasselmeci — APPROVED** [review]
Looks good.

**Comment by brianhlin** [comment]
Wow, a linkchecker that works https://github.com/opensciencegrid/technology/pull/853/checks?check_run_id=1396307192#step:6:17

File: docs/documentation/writing-documentation.md [hunk 1]
```diff
@@ -114,7 +114,8 @@ To contribute content to the OSG, follow these steps to submit a pull request wi
                If `mkdocs.yml` contains does not contain a `nav:` section,
                add the above to the `pages:` section instead.
                This means that the repository is using an older version of MkDocs
-               and will need to be [upgraded to MkDocs v1](#upgrading-to-mkdocs-v1)
+               and will need to be
+               [transitioined to GitHub Actions](publish-osg-pages.md#transitioning-to-github-actions).
```
Comment (matyasselmeci) [inline docs/documentation/writing-documentation.md:118]:
```suggestion
               [transitioned to GitHub Actions](publish-osg-pages.md#transitioning-to-github-actions).
```

**Review by matyasselmeci — APPROVED** [review]
One typo; pre-approving.

File: docs/documentation/writing-documentation.md (hunk 1 above)
Comment (brianhlin) [reply docs/documentation/writing-documentation.md:118]: Man my 'i' key has been causing all sorts of grief lately.

**Comment by brianhlin** [comment]
I believe the PR is failing because it's running the workflow from `master`. Overriding CI and I'll submit a test PR to see if it works for real.

**Comment by brianhlin** [comment]
> I believe the PR is failing because it's running the workflow from master

Whelp, that's not quite right. Something in the html-proofer container env isn't quite right

**Pull request merged**